frontend/src/Nav.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from 'react-router-dom'
+import { useEffect, useState } from 'react';
+
+function Navbar() {
+
+    const [authenticated,setAuthenticated] = useState(false);
+    useEffect(() => {
+        window.ethereum.request({ method: 'eth_accounts' })
+            .then((accounts) => {
+                if(accounts.length > 0) {
+                    setAuthenticated(true);
+                }
+            })
+    }, []);
+
+  if (authenticated){  
```

Comment:
Is there a navbar that theyre supposed to see if theyre not authenticated?

Reply:
No. Didnt think we needed one

Reply:
We do need one.